### PR TITLE
Docs for sc peripheral - state lighting, null texture, extra crafting features

### PR DESCRIPTION
**Pull request by Wojbie**
Adds features added in last few sc-peripherals versions to docs.

Would be grateful for sanity check (and spelling check) on this :D

### CODE REVIEW

File: src/features/sc-peripherals.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ There is an online .3dm to .3dj converter here: <https://3dj.lem.sh/>
 
 #### Example
 
-All arguments except for `shapesOff`, `shapesOn`, `bounds` and `texture` are optional.
+All arguments except for `shapesOff`, `shapesOn`, `bounds` are optional.
```
Comment (scmcgowen) [inline src/features/sc-peripherals.md]: "`shapesOn` and `bounds` "would be more gramatically correct whether or not you decide to add the oxford comma is up to you

**Review by scmcgowen — CHANGES_REQUESTED** [review]

File: src/features/sc-peripherals.md (hunk 1 above)
Comment (Lemmmy) [reply src/features/sc-peripherals.md]:
```suggestion
All arguments except for `shapesOff`, `shapesOn`, and `bounds` are optional.
```

File: src/features/sc-peripherals.md [hunk 2]
```diff
@@ -72,6 +72,7 @@ format.
 - Print models are more efficiently cached, so chunks with lots of identical prints will only bake the print model once
     - Note that this is not the be-all and end-all of performance, there may still be a lot of vertex data to upload to
       the chunk
+- Prints can be crafted with beacon base blocks, honey block and glowstone items for additional features.
```
Comment (Lemmmy) [inline src/features/sc-peripherals.md]:
```suggestion
- Prints can be crafted with beacon base blocks, honey blocks, and glowstone dust or blocks for additional features.
```

**Review by Lemmmy — CHANGES_REQUESTED** [review]
Thanks so much for updating this! Mostly looks good, just a few small edits.

**Pull request merged**